coreteam.py

```diff
@@ -36,7 +36,9 @@
     'boubkerbribri',      # Boubker B.
     'matthieu-rolland',   # Matthieu R.
     'Progi1984',          # Franck L.
-    'Robin-Fischer-PS',   # Robin F
+    'Robin-Fischer-PS',   # Robin F.
+    'samuel-pires',       # Samuel S.
```

Comment:
Samuel S ? Isn't it Samuel P ? 😛

Reply:
```suggestion
    'samuel-pires',       # Samuel P.
```
You are right. My bad.